8data.tex

```diff
@@ -592,18 +592,18 @@
 \end{itemize}
 
 \myparagraph{Observation - Units}
-Units must correspond to the units of body weight, normally in kilograms, (or mean length in cm). The expected value of mean body weight (or mean length) is calculated in a way that incorporates effect of selectivity and retention.
+Units must correspond to the units of body weight, normally in kilograms (kg), (or mean length in centimeters (cm)). The expected value of mean body weight (or mean length) is calculated in a way that incorporates effect of selectivity and retention.
```

Comment:
I don't think that you need to define standard units you can just use them because this is a technical document. At a minimum you should not have the "))" at the end of the sentence.